### PR TITLE
SegDataGenerator Normalization Fix

### PR DESCRIPTION
**Pull request by Richarizardd**
Removing preprocess_input from SegDataGenerator when creating image batch, as the images are already normalized.

Let me know if this change makes sense!

### CODE REVIEW

**Comment by ahundt** [comment]
Could you link to the line where the preprocessing is already done?

**Comment by ahundt** [comment]
@Richarizardd would you mind explaining why this change is necessary?

**Comment by Richarizardd** [comment]
Hi @ahundt , sorry for the delay!

On line 258, you are standardizing the image by calling "self.seg_data_generator.standardize". Thus, I am not sure if it is useful to standardize the image again on line 284 using "preprocess_input" from keras.imagenet_utils.

**Comment by ahundt** [comment]
The standardize function has a bunch of boolean flags, if they're all false no changes should be made when standardize is called. In that case I think the preprocess code would be correct, but I didn't walk through the logic in much detail so if I misunderstood I'd appreciate an additional comment!
